analytics-integrations/taplytics/src/main/java/com/segment/analytics/internal/integrations/TaplyticsIntegration.java

```diff
@@ -0,0 +1,98 @@
+package com.segment.analytics.internal.integrations;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.Traits;
+import com.segment.analytics.ValueMap;
+import com.segment.analytics.internal.AbstractIntegration;
+import com.segment.analytics.internal.model.payloads.GroupPayload;
+import com.segment.analytics.internal.model.payloads.IdentifyPayload;
+import com.segment.analytics.internal.model.payloads.TrackPayload;
+import com.taplytics.sdk.Taplytics;
+import java.util.Map;
+import java.util.Set;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import static com.segment.analytics.internal.Utils.isNullOrEmpty;
+import static com.segment.analytics.internal.Utils.newSet;
+
+/**
+ * Taplytics is a native mobile A/B testing platform that allows you to create new tests and push
+ * them live without changing any code. Analytics, push notifications, and more.
+ *
+ * @see <a href="https://taplytics.com">Taplytics</a>
+ * @see <a href="https://segment.com/docs/integrations/taplytics">Taplytics Integration</a>
+ * @see <a href="https://github.com/taplytics/Taplytics-Android-SDK">Taplytics Android SDK</a>
+ */
+public class TaplyticsIntegration extends AbstractIntegration<Taplytics> {
+  static final String TAPLYTICS_KEY = "Taplytics";
+
+  private static final Set<String> KNOWN_TRAITS =
+      newSet("name", "email", "gender", "firstName", "lastName", "age");
+
+  @Override public void initialize(Analytics analytics, ValueMap settings)
+      throws IllegalStateException {
+    String apiKey = settings.getString("apiKey");
+    Taplytics.startTaplytics(analytics.getApplication(), apiKey);
+  }
+
+  @Override public String key() {
+    return TAPLYTICS_KEY;
+  }
+
+  @Override public void track(TrackPayload track) {
+    super.track(track);
+    JSONObject metaData = track.properties().toJsonObject();
+    Taplytics.logEvent(track.event(), track.properties().value(), metaData);
```

Comment:
The method is an alias to `getDouble()`, if it isn't a number then it returns zero.